lib/gltfPipeline.js

```diff
@@ -61,7 +62,8 @@ function processJSON(gltf, options, callback) {
 
 function processJSONWithExtras(gltfWithExtras, options, callback) {
     var stats = new OptimizationStatistics();
-    addDefaults(gltfWithExtras, options);
+    generateNormals(gltfWithExtras);
```

Comment:
Sorry I wasn't clear with this, but `generateNormals` should go after the initial stages. Probably after `printStats` is good.